src/index.js

```diff
@@ -73,10 +74,15 @@ export const withPropsTable = makeDecorator({
     const content = getStory(context);
     const response = addPropsTable(content, context, mergedOptions);
 
-    channel.emit(
-      'storybook/PropsTable/add_PropsTable',
-      ReactDOM.renderToString(<Story {...response}>{content}</Story>)
-    );
+    const shouldLegacyRender = Boolean(mergedOptions.TableComponent);
```

Comment:
When a user passes in their own TableComponent, we can't feed it through the event emitter (since it's a function). It's why I speculate the code was originally using ReactDOM.renderToString and then emitting.

So, this interactive panel functionality only works for users using the built-in default Table.

Reply:
That's fine. It never worked anyway